docs/posts/hosted.md

```diff
@@ -0,0 +1,75 @@
+# Hosting cellxgene on the web
+
+Cellxgene is intended to be used by researchers on their local machines. However, we recognize that sharing and exploring data on the web is important. We're exploring how we could better support this in the future, and [would welcome your input](https://github.com/chanzuckerberg/cellxgene/issues/875)!
+
+In the meantime, you can see examples of how other groups have approached this [here](gallery). While we don't officially support web deployment, we've offered some guidance below on one way to deploy cellxgene to the web.
+
+### Third-party deployments:
+
+- Chan Zuckerberg BioHub's [Tabula Muris Senis](https://tabula-muris-senis.ds.czbiohub.org/)
+- Novartis's [cellxgene gateway](https://github.com/Novartis/cellxgene-gateway)
+- Interactive Enviroment in the [Galaxy Project](https://galaxyproject.org/) ([patch notes](https://docs.galaxyproject.org/en/release_19.05/releases/19.05_announce.html))
+- Various instances by the Wellcome Sanger Institute
+  - [Tissue Stability Cell Atlas](https://www.tissuestabilitycellatlas.org/)
+  - [Human Kidney Cell Atlas](https://www.kidneycellatlas.org/)
+  - [Atlas of Anopheles hemocytes](https://hemocytes.cellgeni.sanger.ac.uk/)
+  - [Murine melanoma atlas](https://melanoma.cellgeni.sanger.ac.uk/)
+
+# Deploying cellxgene with Heroku
```

Comment:
Like the contribution page, this section of this page may benefit from just linking out instead of restating what is in `experiments/heroku/README.md`